utils/release/lock-master.mjs

```diff
@@ -0,0 +1,28 @@
+import {Octokit} from 'octokit';
+
+const REPO_OWNER = 'coveo';
+const REPO_NAME = 'cli';
+
+export async function changeMasterWriteAccess(canWrite) {
+  const octokit = new Octokit({auth: process.env.GITHUB_CREDENTIALS});
+
+  const currentProtection = await octokit.rest.repos.getBranchProtection({
+    owner: REPO_OWNER,
+    repo: REPO_NAME,
+    branch: 'master',
+  });
+
+  await octokit.rest.repos.updateBranchProtection({
+    owner: REPO_OWNER,
+    repo: REPO_NAME,
+    branch: 'master',
+    ...currentProtection,
+    restrictions: canWrite
+      ? null
+      : {
+          users: [],
+          apps: ['developer-experience-bot'],
```

Comment:
Do you need to add the `[bot]` prefix here?

Reply:
No we're using the appSlug: https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-as-a-github-app
https://github.com/apps/developer-experience-bot